src/style.css

```diff
@@ -0,0 +1,131 @@
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  max-width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
```

Comment:
organize your properties according [css recommendations](https://9elements.com/css-rule-order/)
fix it everywhere